ui/src/org.ts

```diff
@@ -1,17 +1,26 @@
-import * as api from "./api"
+import * as api from "./api";
 import * as avatar from "./avatar";
 import * as event from "./event";
 import * as remote from "./remote";
 import * as validation from "./validation";
 import * as transaction from "./transaction";
-import * as user from "./user"
+import * as user from "./user";
 
 // Types
 export interface Org {
   name: string;
   avatarFallback: avatar.EmojiAvatar;
 }
 
+export interface OrgProject {
```

Comment:
```suggestion
export interface Project {
```

This is already in the `org` module, so can just be the unprefixed version.